src/LiveQuerySubscription.js

```diff
@@ -102,8 +102,11 @@ export default class Subscription extends EventEmitter {
    * @method unsubscribe
    */
   unsubscribe() {
-    var liveQueryClient = CoreManager.getLiveQueryController().getDefaultLiveQueryClient();
-    liveQueryClient.unsubscribe(this);
-    this.emit('close');
+    let _this = this;
```

Comment:
arrow does auto bind but in this case we'll need to distinguish between the promise and the subscription context.